va/validation-authority.go

```diff
@@ -79,6 +99,15 @@ func (va ValidationAuthorityImpl) validateSimpleHTTP(identifier core.AcmeIdentif
 	}
 	hostName := identifier.Value
 
+	// Check for resolver SERVFAIL for A/AAAA records
```

Comment:
This check is unnecessary.  The HTTP(S) query will just fail if there are no A/AAAA records.